workspaces/ui-v2/src/optic-components/pages/diffs/AddEndpointsPage.tsx

```diff
@@ -177,6 +177,8 @@ export function DocumentationRootPageWithPendingEndpoints(props: any) {
                       method: endpoint.method,
                       pathId: endpoint.id,
                     })}
+                    // @nic todo make pendingendpointrow component
+                    purpose=""
```

Comment:
- [ ] lingering todo?

Reply:
This is actually addressed in the follow up PR already https://github.com/opticdev/optic/pull/682 - it's just in there incase there's someone who view this before the linked PR has landed